doc/material-getting-started.md

```diff
@@ -0,0 +1,113 @@
+</p>
+
+The Uno.Toolkit.Material library is available as NuGet packages that can be added to any new or existing Uno solution.
+
+## `MaterialToolkitTheme`
+
+There is now a new, simplified method for initializing the Material Toolkit resources. It is recommended to use the new `MaterialToolkitTheme` going forward since `MaterialToolkitResources` has been deprecated.
+
+### Inheritance
+ResourceDictionary &#8594; MaterialToolkitTheme
+
+### Constructors
+| Constructor    | Description                                           |
+|----------------|-------------------------------------------------------|
+| MaterialToolkitTheme()         | Initializes a new instance of the `MaterialToolkitTheme` `ResourceDictionary`.       |
+| MaterialToolkitTheme(`ResourceDictionary?` colorOverride, `ResourceDictionary?` fontOverride)         | Initializes a new instance of the `MaterialToolkitTheme` `ResourceDictionary` and applies the given overrides       |
+
+### Properties
+
+| Property                  | Type              | Description                                                                                   |
+|---------------------------|-------------------|-----------------------------------------------------------------------------------------------|
+| ColorOverrideSource             | string            | (Optional) Gets or sets a Uniform Resource Identifier that provides the source location of a ResourceDictionary containing overrides for the default Uno.Material Color resources                                            |
+| FontOverrideSource     | string      | (Optional) Gets or sets a Uniform Resource Identifier that provides the source location of a ResourceDictionary containing overrides for the default Uno.Material FontFamily resources            |
+
+## Installation
+
+> [!NOTE]
+> As of [Uno 4.7](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/), the solution template of an Uno app has changed. There is no longer a Shared project (.shproj), it has been replaced with a regular cross-platform library containing all user code files, referred to as the **App Code Library** project. This also implies that package references can be included in a single location without the previous need to include those in all project heads.
+
+
+1. Open an existing Uno project, or create a new Uno project using the `Multi-Platform App (Uno Platform)` template.
+2. In the Solution Explorer panel, right-click on your app's **App Code Library** project (`PROJECT_NAME.csproj`) and select `Manage NuGet Packages...`
+3. Install the [**`Uno.Toolkit.WinUI.Material`**](https://www.nuget.org/packages/Uno.Toolkit.WinUI.Material) package
+4. Add the resources to `AppResources.xaml`:
+	```xml
+	<ResourceDictionary>
+		<ResourceDictionary.MergedDictionaries>
+
+			<!-- Load Material Toolkit resources (also load base Uno.Material resources) -->
+			<MaterialToolkitTheme xmlns="using:Uno.Toolkit.UI" />
```

Comment:
we should clarify whether if MaterialToolkitTheme contains MaterialTheme
like, do the user needs to init both or just the toolkit's one?

Reply:
oh thats supposed to say "also load**s the** base Uno.Material resources" :)

ill make it more clear